ext/drreg/drreg.c

```diff
@@ -2107,80 +2107,26 @@ drreg_event_restore_state_with_ilist(void *drcontext, bool restore_memory,
             last_opcode = instr_get_opcode(inst);
```

Comment:
> This is in preparation for sending the i#4963 fix for review. Split out this harmless refactor to make review easier for that fix.

Add `Issue: #4963`

Reply:
I intentionally omitted it, because this PR isn't strictly related to that issue. It's some code simplification that makes sense even otherwise.

Added the issue# to PR description anyway.